src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,126 @@
+package kitchenpos.application;
+
+import static kitchenpos.common.fixture.MenuFixture.메뉴;
+import static kitchenpos.common.fixture.MenuGroupFixture.메뉴_그룹;
+import static kitchenpos.common.fixture.MenuProductFixture.메뉴_상품;
+import static kitchenpos.common.fixture.ProductFixture.상품;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.List;
+import kitchenpos.common.ServiceTest;
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.dao.ProductDao;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuProduct;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@SuppressWarnings("NonAsciiCharacters")
+@ServiceTest
+class MenuServiceTest {
+
+    private static final BigDecimal PRICE = BigDecimal.valueOf(1).setScale(2, RoundingMode.HALF_UP);
+
+
+    @Autowired
+    private MenuService menuService;
+
+    @Autowired
+    private MenuGroupDao menuGroupDao;
+
+    @Autowired
+    private ProductDao productDao;
+
+    private Long menuGroupId;
+    private MenuProduct menuProduct;
+
+    @BeforeEach
+    void setUp() {
+        menuGroupId = menuGroupDao.save(메뉴_그룹()).getId();
+        Long productId = productDao.save(상품(PRICE)).getId();
+        menuProduct = 메뉴_상품(productId);
+    }
+
+    @Test
+    void 메뉴를_생성한다() {
+        // given
+        Menu menu = 메뉴(menuGroupId, PRICE, List.of(menuProduct));
+
+        // when
+        Menu createdMenu = menuService.create(menu);
+
+        // then
+        assertThat(createdMenu).usingRecursiveComparison()
+                .ignoringFields("id")
+                .ignoringExpectedNullFields()
+                .isEqualTo(메뉴(menuGroupId, List.of(menuProduct)));
```

Comment:
```suggestion
        assertThat(createdMenu).usingRecursiveComparison()
                .ignoringExpectedNullFields()
                .isEqualTo(메뉴(menuGroupId, List.of(menuProduct)));
```

`ignoringFields("id")`는 의도하신건가요?
`ignoringExpectedNullFields()` 때문에 없어도 잘 동작할 것 같아요!

Reply:
넵 해당 테스트에서 id에 관한 내용을 검증하지 않겠다는 것을 분명히 하기 위해 명시해두었습니다! ignoringExpectedNullFields() 는 어떤 field 를 검증하지 않는지 테스트만 보고는 알 수 없어서요.

그럼에도 ignoringExpectedNullFields() 를 함께 두었던 이유는 ignoringFields()의 사용방법을 자세히 알지 못했기 때문이에요. 가령, 해당 테스트에서 Menu 객체가 참조하는 MenuProduct의 seq 필드를 ignoringFields()에 어떻게 작성해야 하는지 몰랐습니다. 

헌데, 이번에 리뷰를 반영하면서 공부했습니다. 내친김에 ignoringExpectedNullFields() 는 모두 제거하겠습니다!